graphql/splunktracing.go

```diff
@@ -23,15 +23,16 @@ func (SplunkTracing) Validate(graphql.ExecutableSchema) error {
 }
 
 func (SplunkTracing) InterceptResponse(ctx context.Context, next graphql.ResponseHandler) *graphql.Response {
-	rc := graphql.GetOperationContext(ctx)
-	if rc == nil {
+	if hasOperationContext := graphql.HasOperationContext(ctx); !hasOperationContext {
 		// There was an invalid operation context, so we can't do anything
 		grip.Critical(message.Fields{
```

Comment:
We shouldn't log critical, because we still log this to the ops-alerts channel (if we do still want to catch this via ops alerts, better to make an alert so we can toggle it or update it without a code change if we need to)